src/inputs/Dropdowns/DropdownBio/DropdownBio.tsx

```diff
@@ -0,0 +1,95 @@
+'use client'
+import styles from './DropdownBio.module.scss'
+import React, { useEffect, useState, useRef } from 'react'
+import 'material-icons/iconfont/material-icons.css'
+import { IoIosArrowUp, IoIosArrowDown, IoIosCheckmark } from 'react-icons/io'
+
+export const DropdownBio = ({
+  label,
+  text,
+  options,
+  onSelect,
+  selectedValue,
+}: {
+  label: string
+  text: string
+  options: string[]
+  onSelect: (option: string) => void
+  selectedValue: string[]
+}) => {
+  const [arrow, setArrow] = useState('IoIosArrowDown')
+  const [isDropdownActive, setDropdownActive] = useState(false)
+
+  const dropdownRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    const handler = (event: MouseEvent) => {
+      if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
+      ) {
+        setDropdownActive(false)
+        setArrow('IoIosArrowDown')
+      }
+    }
+    document.addEventListener('mousedown', handler)
+
+    return () => {
+      document.removeEventListener('mousedown', handler)
+    }
+  }, [])
+
+  const handleDropdown = () => {
+    setArrow(arrow === 'IoIosArrowDown' ? 'IoIosArrowUp' : 'IoIosArrowDown')
+    setDropdownActive(!isDropdownActive)
+  }
+
+  const handleSelect = (option: string) => {
+    onSelect(option)
+  }
+
+  return (
+    <div className={styles.buttonBox}>
+      <div className={styles.label}>{label}</div>
+      <div ref={dropdownRef}>
+        <button onClick={handleDropdown} className={styles.featuresBtn}>
+          {selectedValue.length === 0 ? (
+            <div className={styles.buttonText}>{text}</div>
+          ) : (
+            <div className={styles.buttonTextChecked}>{text}</div>
+          )}
+          <div className={styles.selected}>
+            {selectedValue.length !== 0 && (
+              <div className={styles.selectedCount}>{selectedValue.length}</div>
+            )}
+            {arrow === 'IoIosArrowUp' ? <IoIosArrowUp /> : <IoIosArrowDown />}
+          </div>
+        </button>
+        {isDropdownActive && (
+          <div className={styles.dropdown}>
+            {options.map((option, index) => (
+              <label key={index} className={styles.dropdownInput}>
+                <div
+                  className={`${styles.checkbox} ${
```

Comment:
To dynamic classes, you should use the "bindclasses" library included in the project's documentation.